stl/inc/atomic

```diff
@@ -475,14 +475,33 @@ bool __stdcall _Atomic_wait_compare_non_lock_free(
 
 #ifdef _WIN64
 inline bool __stdcall _Atomic_wait_compare_16_bytes(const void* _Storage, void* _Comparand, size_t, void*) noexcept {
+
+#ifdef _M_ARM64
+#ifndef __clang__

```

Comment:
As this has an `#else` branch, I believe that this should be flipped to avoid the negation. (Our usual convention is to avoid negated tests, if that doesn't complicate the code otherwise).